lib/apple/index.js

```diff
@@ -37,13 +37,30 @@ function parseResult(result) {
 	return {
 		receipt: receipt,
 		/* jshint camelcase:false */
-		transactionId: receipt.transaction_id,
-		productId: receipt.product_id
+		transactionId: getReceiptFieldValue(receipt, "transaction_id"),
+		productId: getReceiptFieldValue(receipt, "product_id")
 		/* jshint camelcase:true */
 	};
 }
 
 
+function getReceiptFieldValue(receipt, field)
+{
```

Comment:
Please try to respect the code style of the project (curly braces at the end of the line).